### DISCUSSION TITLE
higress离线部署问题

### DISCUSSION THREAD

**Discussion by ilovedumplings** [alibaba/higress#1476] [Q&A]
higress/charts/higress-console/values.yaml中的repository: higress-registry.cn-hangzhou.cr.aliyuncs.com/higress是这样的嘛请问?
![image](https://github.com/user-attachments/assets/dc76a384-9497-4ea7-8d52-da8bf2ec281a)

**Comment by johnlanni** [comment]
higress没有提供certmanager的镜像呀，这个问题有点怪🤔️
